rad/src/lib.rs

```diff
@@ -829,6 +841,40 @@ mod tests {
 
     use super::*;
 
+    #[test]
+    fn test_run_http_head_retrieval() {
+        let script_r = Value::Array(vec![
+            Value::Integer(RadonOpCodes::StringParseJSONMap as i128),
+            Value::Array(vec![
+                Value::Integer(RadonOpCodes::MapGetString as i128),
+                Value::Text("etag".to_string()),
+            ]),
+        ]);
+        let packed_script_r = serde_cbor::to_vec(&script_r).unwrap();
+        println!("{:?}", packed_script_r);
+
+        let retrieve = RADRetrieve {
+            kind: RADType::HttpHead,
+            url: "https://witnet.io/_nuxt/img/dragon_reading.a37f8cb.png".to_string(),
```

Comment:
I believe this file will disappear as soon as we make any little change on the website, because Nuxt uses randomized codes in file paths upon rebuilds to deal with caching.

This may be a more future-proof URL: https://en.wikipedia.org/static/images/icons/wikipedia.png

Reply:
Okay. Although this test, and the others on which it's inspired, don't actually perform any HTTP request. They do build a proper a RADRetrieve struct, but then a HTTP response is hardcoded afterwards.